src/map/battle.cpp

```diff
@@ -1386,6 +1407,12 @@ bool battle_status_block_damage(struct block_list *src, struct block_list *targe
 		return false;
 	}
 
+	//TODO: What's causing this status?
+	if (sc->data[SC_HANDICAPSTATE_HOLYFLAME] && flag&BF_MAGIC)	{ 
+			clif_skill_nodamage(0, target, AL_HEAL,damage/2, 1);
```

Comment:
warning C4244: 'argument': conversion from 'int64' to 'int', possible loss of data

Reply:
Works for me.